examples/addsvc/client/main.go

```diff
@@ -143,3 +111,22 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func seed() int64 {
+	b := make([]byte, 8)
+	_, err := rand.Read(b)
+	if err != nil {
+		fmt.Println(err)
+	}
+	return int64(binary.BigEndian.Uint64(b))
```

Comment:
There is only one request made by the client before it shuts down. When using a constant seed, the same endpoint is picked every time, so you cannot have two servers running, launch the client repeatedly and see the requests arriving randomly. Now that I think about it, seeding with a timestamp would be easier, though.